tests/vmi_multus_test.go

```diff
@@ -881,31 +881,42 @@ var _ = Describe("[Serial]SRIOV", func() {
 		It("[test_id:3956]should connect to another machine with sriov interface over IPv4", func() {
 			cidrA := "192.168.1.1/24"
 			cidrB := "192.168.1.2/24"
+			ipA, errA := cidrToIP(cidrA)
```

Comment:
nit: There is no real need for a dedicated error variable, as simple `err` would work as well.
```
ipA, err := cidrToIP(cidrA)
...
ipB, err := cidrToIP(cidrB)
```